Serenity Updates - September - Export with labels (#738)
akeneo/pim-helpcenter: content/updates/2020-09/2-Quick-Export-revamped-UI.md

---
pim_announcement_img: ./img/Quick-export-new-UI_Export-with-labels.png
pim_announcement_alt_img: "Quick Export revamped UI"
pim_announcement_audience:
- CE
- EE
---

# Quick Export revamped UI
::: meta-data type="New" features="Collaboration, Productivity" available="mid September" link-to-doc="../articles/quick-export.html"
:::

> Discover a fresh UI for the Quick Export!

Reviewer: Do you need to go to the next line at the end of this sentence? If not, please put the next sentence on the same line, otherwise, add two spaces at the end of your line.
And remember that the first paragraph will be displayed in the PIM. ;)

This new UI makes it more easy for you to select amongst the existing choices.

To save you time, the selections you've made are stored in your local session. It means the next time you will click on `Quick Export`, the same options will be already selected.

> 

Reviewer: New UI, but no screenshot? So sad...
I guess it's quite mandatory here! ;)

---
pim_announcement_img: ./img/Quick-export-new-UI_Export-with-labels.png
pim_announcement_alt_img: "Quick Export revamped UI"
pim_announcement_audience:
- CE
- EE
---

> # Quick Export revamped UI

Reviewer 2: ```suggestion
# A new user interface for Quick Exports
```

> ::: meta-data type="New" features="Collaboration, Productivity" available="mid September" link-to-doc="../articles/quick-export.html"

Reviewer 2: ```suggestion
::: meta-data type="New" features="Collaboration, Productivity" available="mid-September" link-to-doc="../articles/quick-export.html"
```

:::

> Discover a brand new UI for the Quick Export feature!  

Reviewer 2: ```suggestion
```

> This new UI makes it more easy for you to select amongst the existing choices, especially as we added a new option named "With labels" (see announcement [here](2020-09.html#export-flat-files-with-labels)).

Reviewer 2: ```suggestion
For a better user experience, we created a new user interface to make it easier for you to generate a quick export from the product grid. This improvement follows the addition of a new option for all product exports, a new button called  "With labels" (see announcement [here](2020-09.html#export-flat-files-with-labels)).
```

> To save you time, the selections you've made are stored in your local session. It means the next time you will click on `Quick Export`, the same options will be already selected.

Reviewer 2: ```suggestion
To save you time, the custom choices you make for your XLSX or CSV quick export are saved. So next time you click on `Quick Export`, the last export configuration will be displayed.  
```
